### DISCUSSION TITLE
HIST command not working (Hero11, 12 and 13) - "QRCode non-GoPro"

### DISCUSSION THREAD

**Discussion by BlackBird11** [gopro/labs#1206] [Q&A] [answered]
Like the title says none of my GoPros seem to accept the command HIST=1

I asked your ChatGPT bot and got: HIST=1HSTP=5,24,50
This is also not working.

On my Hero12 I have it running since a few months, but I can't disable it with HIST=0. What am I doing wrong?

Other commands i have running:
*OWNR="xxxxxxx" !MEXPS=1 !MBITR=190 !M24HZ=1 !MEXPN=8000 !MEXPX=50 !MNR01=1 !MLOGB=400

Thanks in advance!

![IMG_6771](https://github.com/user-attachments/assets/db6f3f58-41be-48f0-a82b-043f98fe0fff)

**Comment by dnewman-gpsw** [comment, marked as answer]
HIST requires the same preamble either !M or * on newer cameras.  Sorry the chatbot is not yet perfect.

Documentation on the command is here: https://gopro.github.io/labs/control/tech/

**Reply by BlackBird11** [reply]
My bad, couldn't find the answer anywhere, thanks for clarifying!
